services/movie_session.py

```diff
@@ -42,3 +43,10 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
Consider handling exceptions for the `get` method in `delete_movie_session_by_id` to manage cases where a movie session with the given ID does not exist. You can use `MovieSession.DoesNotExist` to catch such exceptions and handle them appropriately.